src/components/app-header/app-header.js

```diff
@@ -2,20 +2,53 @@ import React, { Component } from "react";
 import './app-header.css';
 
 export default class AppHeader extends Component {
-    
+
+  constructor() {
+    super();
+
+    this.state = {
+      text: ''
+    };
+
+    this.addNoteText = (e) => {
+      this.setState({
+        text: e.target.value
+      });
+    };
+
+    this.onSubmit = (e) => {
+      e.preventDefault();
+      this.props.onNoteAdded(this.state.text);
+    };
+
+  }
+
     render() {
+         
         return (
-        <div>
-            <header className="app-header">
-            <input className='search-form'
-            placeholder='Type to search'></input>
-            <input className='type-box'
-            placeholder='Write a note...'></input>
-            <button className='add-note-btn'>
-              Add note
+        <form onSubmit={this.onSubmit}>
```

Comment:
[form](https://www.w3schools.com/html/html_forms.asp) - is usually a data which is sent on server, probably div is better here